cost-analyzer/values.yaml

```diff
@@ -130,6 +130,12 @@ saml: # enterprise key required to use
         assertionvalues:
           - "readonly"
 
+systemProxy:
```

Comment:
Can we reference docs or add comments?

Reply:
I've added comments on the behavior (and I've also tested all four cases to ensure correct behavior), i.e., 
1. systemProxy.enabled = false, systemProxy.useHttps = false -- doesn't show up in template
2. systemProxy.enabled = false, systemProxy.useHttps = true -- doesn't show up
3. systemProxy.enabled = true, systemProxy.useHttps = false -- HTTP_PROXY is set
4. systemProxy.enabled = true, systemProxy.useHttps = true -- HTTPS_PROXY is set

let me know if Bob has any feedback to share